src/components/CopyToClipboard/CopyToClipboard.tsx

```diff
@@ -1,29 +1,42 @@
-import { ClipboardCopy, Split, SplitItem } from '@patternfly/react-core';
+import { ClipboardCopy, ClipboardCopyAction, Split, SplitItem } from '@patternfly/react-core';
+
+import styles from './CopyToClipboard.module.css';
 
 interface ICopyToClipboardProps {
   prefixComponent?: React.ReactNode;
   suffixComponent?: React.ReactNode;
+  isInline?: boolean;
 }
 
 /**
  * CopyToClipboard item to display customized ClipboardCopy component.
  *
  * @param prefixComponent - any component before ClipboardCopy
  * @param suffixComponent - any component after ClipboardCopy
+ * @param isInline - whether to use inline style for the component
  * @param children - the content to be used for ClipboardCopy
  */
 export const CopyToClipboard = ({
   prefixComponent,
   suffixComponent,
+  isInline,
   children,
 }: React.PropsWithChildren<ICopyToClipboardProps>) => (
   <Split hasGutter>
     {prefixComponent && <SplitItem>{prefixComponent}</SplitItem>}
     <SplitItem isFilled>
-      <ClipboardCopy removeFindDomNode isReadOnly hoverTip="Copy" clickTip="Copied">
+      <ClipboardCopy
+        removeFindDomNode
+        isReadOnly
+        hoverTip="Copy"
+        clickTip="Copied"
+        variant={isInline ? 'inline-compact' : 'inline'}
+        additionalActions={<ClipboardCopyAction>{suffixComponent}</ClipboardCopyAction>}
```

Comment:
Read the code carefully, this is the `ClipboardCopy` component provided by PatternFly, not our own component. `additionalActions` is the API from PatternFly `ClipboardCopy` component to enable any icon after the copy icon, and I already reused `suffixComponent` so from the line 15 you can see we have no `additionalActions` parameter required any more.

Reply:
yes, actually I understand it all correctly, but I still think using `additionalActions` in `ClipboardCopy` should not be needed, and using your suffix should be preferred in both cases

Reply:
I don't think this would be what you want. Not using `additionalActions` will make the icon display outside the component, and lose consistency. See screenshot:
![image](https://user-images.githubusercontent.com/6870455/226204444-81f66db0-2fab-4011-a92d-c86f1db15d5c.png)

When using `additionalActions` it looks good as following:
![image](https://user-images.githubusercontent.com/6870455/226204549-6272e28f-0638-465b-a681-131d43876600.png)

Reply:
It's because you are still applying block stylesheets around it. But anyway, feel free to keep it.